File: ios/Classes/enums/GroupTipsType.swift
import ImSDK

//  Created by 蒋具宏 on 2020/2/15.
//  群提示类型
public enum GroupTipsType : Int{
    case Invalid = 0
    case Join = 1
    case Quit = 2
    case Kick = 3
    case SetAdmin = 4
    case CancelAdmin = 5
    case ModifyGroupInfo = 6
    case ModifyMemberInfo = 7
    case AaddGroup = 8
    case DelGroup = 9
    
    /**
     * 根据腾讯云获得枚举
     */
    static func getByTIM_GROUP_TIPS_TYPE(type : TIM_GROUP_TIPS_TYPE)->GroupTipsType{
        return (GroupTipsType(rawValue: type.rawValue) ?? nil)!;
    }
}

File: lib/enums/group_tips_type.dart
/// 群组事件通知类型
enum GroupTipsType {
  /// 修改成员信息
  AaddGroup,

  /// 取消管理员
  CancelAdmin,

  /// 修改成员信息
  DelGroup,

  /// 非法
  Invalid,

  /// 加入群组
  Join,

  /// 被踢出群组
  Kick,

  /// 修改群资料
  ModifyGroupInfo,

  /// 修改成员信息
  ModifyMemberInfo,

  /// 主动退出群组
  Quit,

  /// 设置管理员
  SetAdmin,
}
